smallville/src/main/java/io/github/nickm980/smallville/entities/memory/Plan.java

```diff
@@ -18,17 +18,20 @@ public class Plan extends Memory implements TemporalMemory, NaturalLanguageConve
 
     private final LocalDateTime time;
     private boolean isShortTerm;
+
+    private SimulationTime timekeeper;
     public PlanType type;
 
-    public Plan(String description, LocalDateTime time) {
-	this(description, time, false);
+    public Plan(String description, LocalDateTime time, SimulationTime timekeeper) {
```

Comment:
Should probably get rid of LocalDateTime and handle all time logic in SimulationTime